src/main/kotlin/no/nav/syfo/persistering/db/PersisterManuellOppgaveQueries.kt

```diff
@@ -16,19 +19,21 @@ fun DatabaseInterface.opprettManuellOppgave(papirSmRegistering: PapirSmRegisteri
                 dokument_info_id,
                 dato_opprettet,
                 oppgave_id,
-                ferdigstilt
+                ferdigstilt,
+                papir_sm_registrering
                 )
-            VALUES  (?, ?, ?, ?, ?, ?, ?, ?)
+            VALUES  (?, ?, ?, ?, ?, ?, ?, ?, ?)
             """
         ).use {
             it.setString(1, papirSmRegistering.sykmeldingId)
             it.setString(2, papirSmRegistering.journalpostId)
             it.setString(3, papirSmRegistering.fnr)
             it.setString(4, papirSmRegistering.aktorId)
             it.setString(5, papirSmRegistering.dokumentInfoId)
-            it.setTimestamp(6, Timestamp.valueOf(papirSmRegistering.datoOpprettet))
+            it.setTimestamp(6, Timestamp.valueOf(papirSmRegistering.datoOpprettet?.atZoneSameInstant(ZoneId.of("UTC"))?.toLocalDateTime()))
```

Comment:
Siden denne er OffsetDateTime fra før mener jeg du kan gjøre sånn som dette: 
`it.setTimestamp(6, Timestamp.from(papirSmRegistrering.datoOpprettet.toInstant()))`

(med nullsjekk da)